helpers/bandchain2.js/src/obi.ts

```diff
@@ -0,0 +1,198 @@
+abstract class ObiBase {
+  abstract encode(value: any): Buffer
+  abstract decode(buff: Buffer): any[]
+}
+
+export class ObiInteger extends ObiBase {
+  static REGEX = /^(u|i)(8|16|32|64|128|256)$/
+  isSigned: boolean
+  sizeInBytes: number
+
+  constructor(schema: string) {
+    super()
+    this.isSigned = schema[0] === 'i'
+    this.sizeInBytes = parseInt(schema.slice(1)) / 8
+  }
+
+  encode(value: BigInt) {
+    let newValue = BigInt(value)
+    return Buffer.from(
+      [...Array(this.sizeInBytes)]
+        .map(() => {
+          const dataByte = newValue % BigInt(1 << 8)
+          newValue /= BigInt(1 << 8)
+          return Number(dataByte)
+        })
+        .reverse(),
+    )
+  }
```

Comment:
Set the return type on every function?

Could you check all functions in this file?